e2e-tests/package.json

```diff
@@ -8,7 +8,7 @@
   "author": "janus-idp",
   "license": "Apache-2.0",
   "scripts": {
-    "test": "playwright test",
+    "test": "playwright test --reporter=list",
```

Comment:
Using a hardcoded reporter configuration in your package.json, such as "test": "playwright test --reporter=list", might seem straightforward but it introduces several limitations that could affect the flexibility and scalability of your test execution strategy. This approach restricts you to a single type of report output for all test runs, making it cumbersome to switch between different reporters (e.g., html, allure, json, list) based on varying requirements or contexts.

A more dynamic approach involves specifying the reporter directly in the command line when executing your tests, like so: npx playwright test --reporter=json. This method allows you to easily switch between different reporter outputs without needing to modify your package.json or test configuration files.  It enhances flexibility by enabling you to tailor the report output to specific needs, such as generating JSON reports for CI pipelines and HTML reports for local development and review.